data/scripts.js

```diff
@@ -140,7 +140,10 @@ exports.BattleScripts = {
 				this.add('-notarget');
 				return true;
 			}
-			if (targets.length > 1) move.spreadHit = true;
+			if (targets.length > 1) {
+				this.attrLastMove('[spread]');
+				move.spreadHit = true;
+			}
```

Comment:
I would actually just use the target's location ID (`p1a`, `p1b`, etc) so we could use `,` as the separator.